web/modules/custom/portland/portland.module

```diff
@@ -824,6 +824,17 @@ function _validate_document(&$form, \Drupal\Core\Form\FormStateInterface $form_s
       $efiles_link = $form_state->getValue('field_efiles_link');
       $document = $form_state->getValue('field_document');
     }
+
+    // PGOV-809 Validate document and efiles link fields
+    if( ( empty($efiles_link[0]["uri"]) && empty($document[0]["fids"]) ) ) {
```

Comment:
It looks like there's an extra set of parentheses around the if statement. (Not really critical, but could be cleaned up a bit.)

Reply:
Good catch! Extra parentheses removed.